apps/fluent-tester/src/FluentTester/theme/applyTheme.ts

```diff
@@ -0,0 +1,78 @@
+import { OfficePalette, PartialTheme, Theme } from '@fluentui-react-native/framework';
+import { createPartialOfficeTheme, getThemingModule } from '@fluentui-react-native/win32-theme';
+import { testerTheme } from './CustomThemes';
+
+export type ThemeNames = 'Fluent' | 'Office' | 'Caterpillar';
+
+export const themeChoices = [
+  { label: 'Fluent', value: 'Fluent' },
+  { label: 'Office', value: 'Office' },
+  { label: 'Caterpillar', value: 'Caterpillar' },
+];
+
+export function applyCaterpillarTheme(parent: Theme): PartialTheme {
+  return parent.host?.appearance === 'dark'
+    ? {
+        colors: {
+          buttonBackground: '#111111',
+          buttonBackgroundHovered: '#ffcd11',
+          buttonBackgroundPressed: '#eeeeee',
+          buttonText: '#ffffff',
+          buttonTextPressed: '#111111',
+          buttonTextHovered: '#000000',
+        },
+        components: {
+          Button: {
+            borderWidth: 0,
+            tokens: {
+              borderWidth: 0,
+            },
+          },
+        },
+      }
+    : {
+        colors: {
+          buttonBackground: '#ffcd11',
+          buttonBackgroundHovered: '#111111',
+          buttonBackgroundPressed: '#eeeeee',
+          buttonText: '#000000',
+          buttonTextPressed: '#111111',
+          buttonTextHovered: '#ffffff',
+        },
+        components: {
+          Button: {
+            borderWidth: 0,
+            tokens: {
+              borderWidth: 0,
+            },
+          },
+        },
+      };
+}
+
+let currentTheme: ThemeNames = 'Fluent';
+
+const themingModule = getThemingModule()[0];
+const officeBase = themingModule
```

Comment:
Same question as before that this seems odd to have Office specific code in an open source framework...